src/include/rear/ui.rb

```diff
@@ -68,6 +68,23 @@ def UsedModules
       deep_copy(modules)
     end
 
+    # returns a list of mountpoints that have to be included in BACKUP_PROG_INCLUDE
+    def GetMountpoints
+      # -D excludes pseudo filesystems
+      # -r raw output, no ascii-art
+      # -n no column header
+      cmd = "findmnt -D -t notmpfs,nodevtmpfs -o TARGET -n -r | sort"
+      output = SCR.Execute(path(".target.bash_output"), cmd, "")
+      mountpoints = output["stdout"].split(/\n/)
+
+      # Do not include /, /.snapshots and /var/crash
+      mountpoints.delete_if { |e|
+        %w(/ /.snapshots /var/crash).include?(e)
+      }
```

Comment:
`mountpoints -= ["/", "/.snapshots", "/var/crash"]`
and for documentation, you do not need to write what code did, but why. Why these three have to be excluded?